### PR TITLE
hp/elitebook/2560p: init

### PR DESCRIPTION
**Pull request by aviallon**
First HP device? Interesting.

Add some required tweaks to make things work properly, and some toggle-able config for the iGPU.

### CODE REVIEW

File: hp/elitebook/2560p/igpu.nix [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{ config, pkgs, lib, ... }:
+with lib;
+let
+  xcfg = config.services.xserver;
+  cfg = config.hardware.hp.elitebook.graphics;
+in
+{
```
Comment (Mic92) [inline hp/elitebook/2560p/igpu.nix:7]: Also I would be interested why you want to toggle the iGPU? Does the board also comes with dedicated graphic cards?

File: hp/elitebook/2560p/default.nix [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{ config, lib, pkgs, ... }:
+with lib;
+let
+  xcfg = config.services.xserver;
+in
+{
+  imports = [
+    ../../../common/cpu/intel
```
Comment (Mic92) [inline hp/elitebook/2560p/default.nix:8]: By including this profile, you are currently enabling intel graphics unconditionally.

File: hp/elitebook/2560p/igpu.nix (hunk 1 above)
Comment (aviallon) [reply hp/elitebook/2560p/igpu.nix:7]: Actualy, I don't want to toggle the iGPU, but only the hardware acceleration (vaapi, etc).
Someone might want to disable it for some reason.

File: hp/elitebook/2560p/default.nix (hunk 2 above)
Comment (aviallon) [reply hp/elitebook/2560p/default.nix:8]: See above comment.

File: hp/elitebook/2560p/igpu.nix (hunk 1 above)
Comment (aviallon) [reply hp/elitebook/2560p/igpu.nix:7]: Oh wait, I'm stupid.

**Comment by Mic92** [comment]
fixed in https://github.com/NixOS/nixos-hardware/pull/371

**Pull request closed without merging**